worlds/stardew_valley/stardew_rule/rule_explain.py

```diff
@@ -25,19 +57,25 @@ def __post_init__(self):
             self.sub_rules = []
 
     def summary(self, depth=0) -> str:
+        if self.mode is ExplainMode.CLIENT:
+            depth *= 2
+
         summary = "  " * depth + f"{str(self.rule)} -> {self.result}"
```

Comment:
Somehow it makes me a bit uneasy to name the variable the same as the method, with the same casing and all, and Python's lack of formal declaration. Is this considered okay?